modules/theme-tailwind/molecules/Newsletter.vue

```diff
@@ -45,43 +45,43 @@ import SolidTwitter from '#ioc/icons/SolidTwitter'
 import Button from '#ioc/atoms/Button'
 import useI18n from '#ioc/composables/useI18n'
 import useSubscribeEmailToNewsletter from '#ioc/services/useSubscribeToNewsletter'
-import useShowErrorNotification from '#ioc/composables/useShowErrorNotification'
 import useShowSuccessNotification from '#ioc/composables/useShowSuccessNotification'
 import { ref } from 'vue'
 
 const { t } = useI18n()
 const loading = ref(false)
 const subscribed = ref(false)
-const showErrorNotification = useShowErrorNotification()
 const showSuccessNotification = useShowSuccessNotification()
 const subscribeToNewsletter = useSubscribeEmailToNewsletter()
 
 const onSubmitNewsletter = async ({ newsletter }: { newsletter: string }) => {
-  try {
-    loading.value = true
-
-    const { statusText, ok } = await subscribeToNewsletter(newsletter)
-
-    loading.value = false
+  loading.value = true
 
-    if (!ok) {
-      showErrorNotification(new Error(statusText))
-    } else {
-      showSuccessNotification(statusText ? t(`${statusText}`) : t(`Subscribed`), t('Thank you.'))
+  try {
+    const { status } = await subscribeToNewsletter(newsletter)
 
-      subscribed.value = true
+    if (status === 'SUBSCRIBED') {
+      showSuccessNotification(t('Subscribed'), t('Thank you.'))
+    } else if (status === 'NOT_ACTIVE') {
+      showSuccessNotification(
+        t('Requires a confirmation'),
+        t('Please, confirm your sign-up by link from e-mail we sent to your e-mail address.'),
+      )
     }
-  } catch (error) {
-    console.warn(error)
+    subscribed.value = true
+  } finally {
+    loading.value = false
   }
 }
 </script>
 
 <i18n lang="yaml">
 cs-CZ:
```

Comment:
separate eng and cz by global key like that 
en-US:
  thankYou: Thank you.
cs-CZ:
  thankYou: Děkujeme.